nengo_gui/components/sim_control.py

```diff
@@ -181,7 +181,8 @@ def javascript(self):
         fn = json.dumps(self.page.filename)
         js = self.javascript_config(info)
         return ('sim = new Nengo.SimControl(control, %s);\n'
-                'toolbar = new Nengo.Toolbar(%s); ' % (js, fn))
+                'toolbar = new Nengo.Toolbar(%s); '
```

Comment:
Probably want a `\n` at the end of the second line (not that it will matter to JS but if someone is inspecting the JS returned it might help)